docs/Makefile

```diff
@@ -7,8 +7,11 @@ pull:
 	docker pull $(IMAGE)
 
 .PHONY: docs
-docs: pull
+docs: check-cloudwatch-integration pull
 	docker run --init -v $(shell pwd)/sources:$(CONTENT_PATH):Z -p $(PORT) --rm -it $(IMAGE)
 
+check-cloudwatch-integration:
```

Comment:
the exporter includes shipped in the binary a list of supported services. To keep the docs in sync, created a small CLI that generates the `supported services` section and checks the doc md file.